pkg/ocm/versions.go

```diff
@@ -322,8 +322,9 @@ func CheckAndParseVersion(availableUpgrades []string, version string) (string, e
 	if err != nil {
 		return "", err
 	}
+	isPreRelease := a.Prerelease() != ""
 	versionSplit := a.Segments64()
-	if versionSplit[2] > 0 {
+	if len(versionSplit) > 2 && versionSplit[2] > 0 || (versionSplit[2] == 0 && isPreRelease) {
```

Comment:
I'm not sure I fully understand -- is a pre release version just `X.X.0`?

Reply:
They have a -rc which gets caught under the isPrerelease. I've taken a look at other channel groups (fast, nightly and candidate) only fast has versions where z from x.y.z are different than 0. So that should be okay